Source/PartnerCenter.CustomerPortal/BusinessLogic/Utility/ApiClient.cs

```diff
@@ -0,0 +1,44 @@
+﻿// -----------------------------------------------------------------------
```

Comment:
We should be initializing a new instance of the HttpClient every time we perform an operation. I would recommend that we create a static instance of it either for this class or globally in the Application Domain. Check out 
https://docs.microsoft.com/en-us/azure/architecture/antipatterns/improper-instantiation/ for more details as to why I am making this recommendation.